sys/include/net/gnrc/ipv6/nc.h

```diff
@@ -143,6 +146,15 @@ typedef struct {
      */
     vtimer_t nbr_adv_timer;
 
+#ifdef MODULE_GNRC_SIXLOWPAN_ND
+    vtimer_t rtr_sol_timer; /**< Retransmission timer for unicast router solicitations */
+#endif
+#ifdef MODULE_GNRC_SIXLOWPAN_ND_ROUTER
+    vtimer_t type_timeout;                  /**< Timer for type transissions */
+    eui64_t eui64;                          /**< the unique EUI-64 of the neighbor (might be
```

Comment:
Seems to be like a waste of memory, to store l2_addr (and l2_addr_len) plus the eui64. Shouldn't 6lowpan-nd **save** memory?